Lombiq.BaseTheme/Models/ZoneDescriptor.cs

```diff
@@ -16,18 +16,30 @@ public class ZoneDescriptor
     public const string LayoutElementClassName = "layoutElement";
     public const string LeafClassName = LayoutElementClassName + "_leaf";
 
+    // Elements that may be zones and are landmarks, see https://html-validate.org/rules/unique-landmark.html.
+    private static readonly string[] _landmarkElements = ["aside", "footer", "form", "header", "main", "nav", "section"];
```

Comment:
You can use the constants that already exist in the `AngleSharp.Dom` namespace (since OC depends on AngleSharp anyway). Also might as well keep it short:
```csharp
using static AngleSharp.Dom.TagNames;

private static readonly string[] _landmarkElements = [Aside, Footer, Form, Header, Main, Nav, Section];
```

Also please use these constants [here](https://github.com/Lombiq/Orchard-Base-Theme/pull/110/files#diff-c2a9a15bcd88adec4c18edbdaf07e974e549a4aee64e9bcf2009d8309e079356R84) and [here](https://github.com/Lombiq/Orchard-Base-Theme/pull/110/files#diff-c2a9a15bcd88adec4c18edbdaf07e974e549a4aee64e9bcf2009d8309e079356R90) as well.

Reply:
Thanks for the code sample!